packages/atomic/src/components/commerce/atomic-commerce-load-more-products/e2e/fixture.ts

```diff
@@ -0,0 +1,28 @@
+import {test as base} from '@playwright/test';
+import {
+  AxeFixture,
+  makeAxeBuilder,
+} from '../../../../../playwright-utils/base-fixture';
+import {SearchBoxPageObject as SearchBox} from '../../atomic-commerce-search-box/e2e/page-object';
+import {AtomicCommerceProductsPageObject as Products} from '../../atomic-product/e2e/page-object';
+import {LoadMoreProductsPageObject as LoadMore} from './page-object';
+
+type MyFixtures = {
```

Comment:
Total pet peeve, but can we rename this to "AtomicFixtures" or something? MyFixtures sounds silly 😅